File: csharp/01_variables.cs
using System;

class Hello{
    public static void Main(){

        string name = "Boris";
        int age = 30;
        double height = 6.1;
        char grade = 'A';
	    
	Console.WriteLine("There was a man called {0}.", name);
    	Console.WriteLine("He was {0} years old.", age);
    	Console.Write("He was {0} foot tall and ", height);
    	Console.WriteLine("Scored an {0} on the test.", grade);
    }
}

File: c/01_variables.c
#include <stdio.h>

int main(){
    char name[100] = "Boris";
    int age = 30;
    float height = 6.1;
    
    printf("There was a man called %s.\n", name);
    printf("He was %i years old.\n", age);
    printf("He was %.1f foot tall.\n", height);
    
    return 0;
}
